vnext/Scripts/Microsoft.ReactNative.Managed.SignConfig.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<SignConfigXML>
+  <job platform="" configuration="" dest="__INPATHROOT__" jobname="microsoftreactnative" approvers="">
+    <!-- Overwrite the input files to simplify downstream things like nuget package generation. -->
+
+    <!-- Is Authenticode right here? Do we need strong-signing for C# dlls? -->
```

Comment:
I'm on the fence...
One one side: This is now netcoreapp3.1 so Authenticode is enough. 
See [Strong Name Signing](https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/docs/project/strong-name-signing.md) doc on the netcore runtime page.
On the other side: If we ever have a customer for managed code, that requires strong name signing, adding it in a future version would be a breaking change.

@jonthysell: Do you know if we can create a strongname for the .winmd from Microsoft.ReactNative as well, else we can't even strongname this asembly...

Reply:
I get that the codesign tool might not need it because it's a netcoreapp, but isn't Microsoft.ReactNative.Managed.dll just another C# UWP dll that the app/native module builds against?